src/umockdev-record.vala

```diff
@@ -444,6 +478,7 @@ main (string[] args)
                             null, out child_pid);
     } catch (Error e) {
             stderr.printf ("Cannot run %s: %s\n", opt_devices[0], e.message);
+            remove_dir (root_dir);
```

Comment:
Is it really that important to clean up in this corner case? As C/glib makes it so excessively annoying to recursively remove a directory, let's just not bother?

Reply:
Eh, not a big deal either, considering we have the helper already?